IQDropDownTextField Demo/Drop Down TextField/Drop Down TextField-Info.plist

```diff
@@ -21,9 +21,13 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1.0.9</string>
+	<string>1.1.1</string>
```

Comment:
You should not increase build version of Info.plist file, could you please revert this change?